bochs/config.cc

```diff
@@ -62,6 +62,8 @@ int bochsrc_include_level = 0;
 
 extern bx_debug_t bx_dbg;
 
+bool port_e9_hack_all_rings;
```

Comment:
the config.cc is not ideal location and any other location is not ideal.
we are doing Bochs param tree exactly this stuff.
see for example for how MWAIT_IS_NOP handled.

You create new bx_param_bool_c with the value e9_all_ports and set it in config.cc
When read in io.cc

Reply:
done!